custom/opm/tests/case_reports.py

```diff
@@ -7,11 +7,11 @@
 
 from casexml.apps.case.models import CommCareCase
 from couchforms.models import XFormInstance
-from ..reports import SharedDataProvider
 from dimagi.utils.dates import DateSpan, add_months
 
 from ..beneficiary import OPMCaseRow
-from ..reports import CaseReportMixin
+from ..constants import *
```

Comment:
i agree import * is not great here. referencing things like `constants.CFU1_XMLNS` is a nice workaround